### PR TITLE
Added management command for creating Tiers/TierPrograms

### LINKED ISSUE #1125: add Tiers n stuff to management command
it would be cool if the management command for fake data (`gen_realistic_search_data`) also set up financial aid tiers and whatnot on one of the newly created programs.

### PR DESCRIPTION
**Pull request by alvinsiu**
#### What are the relevant tickets?
Closes #1125 

#### What's this PR do?
Creates the "create_tier" management command for generating financial aid Tiers and TierPrograms.

### CODE REVIEW

**Comment by aliceriot** [comment]
Functionality looks good! reading over the code now.

File: financialaid/management/commands/create_tiers.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        if not options["add_to_existing"]:
+            # Filter to just those that don't have any TierPrograms
+            programs = programs.filter(tier_programs__isnull=True)
+
+        tiers_created = 0
+        programs_with_tiers_created = 0
+        for program in programs:
+            # Create TierPrograms
+            number_remaining = int(options["tiers"]) - program.tier_programs.count()
+            if number_remaining <= 0:
```
Comment (aliceriot) [inline financialaid/management/commands/create_tiers.py]: `number_remaining` sounds to me like a counter we're going to decrement, maybe `tiers_to_create_count` or something along those lines would be more clear.

Comment (alvinsiu) [inline financialaid/management/commands/create_tiers.py]: Sure, I'll make that change now!

**Comment by alvinsiu** [comment]
Changes made!

**Pull request merged**